cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -94,9 +106,39 @@ var Plugin = new Class({
      */
     _setupUI: function setupUI(container) {
         var wrapper = $('.' + container);
+        var contents;
+
+        if (wrapper.length > 1) {
```

Comment:
not really. all of this is synchronous and is blocking the main thread anyways
in 99.9999% of the cases it won't be noticeable